webf/lib/src/html/head.dart

```diff
@@ -261,6 +262,13 @@ class StyleElement extends Element {
     return ret;
   }
 
+  @override
+  void childrenChanged() {
```

Comment:
no needs for this code

Reply:
ownerDocument.updateStyleIfNeeded() has bean called in recalculateStyle function,  so delete it in the chidrenChanged of StyleElement.